examples/ppo/ppo_lib.py

```diff
@@ -274,31 +296,39 @@ def train(
   log_frequency = 40
   checkpoint_frequency = 500
 
-
-  for s in range(loop_steps):
+  initial_params = get_initial_params(jax.random.PRNGKey(0), model)
+  state = create_train_state(initial_params, model, config,
+                             loop_steps * config.num_epochs)
```

Comment:
I'm not sure I follow how steps are counted:
I think below you say that `state.step = s * num_epochs * num_agents * actor_steps / batch_size`.
So `max(state.step) = loop_steps * num_epochs * num_agents * actor_steps / batch_size`.
But here you say `train_steps = loop_steps * num_epochs` - so that seems to be missing the `num_agents * actor_steps / batch_size` part.

Reply:
You're right, fixed